src/Apps/Collect/Components/Collection/Header/__tests__/DefaultHeader.test.tsx

```diff
@@ -28,15 +28,15 @@ describe("artworks", () => {
 
     expect(hasDuplicateArtworks(headerArtworks)).toBeTruthy()
     expect(headerArtworks.length).toBeGreaterThanOrEqual(artworks.length)
-    expect(headerArtworks).toHaveLength(3)
+    expect(headerArtworks).toHaveLength(4)
   })
 
   it("returns only the number of artworks necessary to fill the header", () => {
     const artworks = defaultCollectionHeaderArtworks.hits
 
     const headerArtworks = getHeaderArtworks(artworks, 1375, false)
-    expect(artworks.length).toBeGreaterThan(headerArtworks.length)
-    expect(headerArtworks).toHaveLength(7)
+    expect(artworks.length).toBeGreaterThanOrEqual(headerArtworks.length)
+    expect(headerArtworks).toHaveLength(10)
```

Comment:
It's not obvious to me why this test would have gone from requiring 7 images to fill the header to requiring 10. Can you help me understand? I would have expected this not to change, since we're only placing as many images as it takes to fill the header.